wagtail/test/settings.py

```diff
@@ -183,6 +183,10 @@
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 
+# HTML email notifications are False by default, however we test with HTML email
+# notifications enabled to cover HTML email code paths
+WAGTAILADMIN_NOTIFICATION_USE_HTML = True
```

Comment:
Instead of changing the settings, I think we can subclass the test cases with this setting turned on.